packages/filesystem/filesystem.spec

```diff
@@ -55,6 +56,7 @@ ln -s lib %{buildroot}%{_cross_prefix}/lib64
 /dev
 /proc
 /root
+/root/.aws
```

Comment:
nit: this is technically included by `/root` above, the same way `/media` includes `/media/cdrom` 
```suggestion
```

Reply:
Without this line there is an error writing to the subdirectory that it is read only. I believe the subdirectory needs to be explicitly created in order for the mount operation to succeed.

Reply:
> Without this line there is an error writing to the subdirectory that it is read only.

An error during the rpm build or at runtime? There shouldn't be a need to write to `/root` as long as the directory exists and is included in the package. Same as with `/media/cdrom` which is only used as a mount point at runtime, and only included implicitly because of the `/media` line.

Reply:
Error on boot:

```
[FAILED] Failed to mount AWS configuration directory (/root/.aws).
...
thar-be-settings[1497]: Failed to write template directory to disk at /root/.aws: Read-only file system (os error 30)
```

Reply:
I was able to clean this up a bit in https://github.com/bottlerocket-os/bottlerocket/pull/2542 - PTAL.